packages/fluentui/perf-test-northstar/just.config.ts

```diff
@@ -33,6 +34,13 @@ task('perf-test:run', () => {
   return runPerfTest(argv().base);
 });
 
+task('check-if-package-affected', () => {
+  const affected = getAffectedPackages().has('@fluentui/react-northstar');
```

Comment:
This task needs configuration of some sort it seems wrong to hardcode northstar  and only check that.

Is it possible to accept a CLI arg in just ?